<tr><td colspan="4"><pre>@@ -29,4 +29,5 @@
 * [[Data hierarchy|Hierarchies]]
 * Observation data
+* [[Dark data]]
 * [[Market data|Trade and market data]]
 
@@ -168,4 +169,7 @@
 * Ataccama
 * Precisely
+}}&lt;noinclude>{{documentation|content=
+A [[template:navbox|navbox]] for [[data]]-related topics.
 
-}}
+[[Category:Computing templates]]
+}}&lt;/noinclude>
</pre></td></tr>